README.md

```diff
@@ -136,7 +136,7 @@ which is nearly identical to the above example. However, there is no build.rs sc
 so in place of the `include!` macro, one finds the procedural macro:
 
 ```rust
-dtrace_provider!("probe-test-macro/test.d");
```

Comment:
It seems like a good idea to put a note in the README about how this file is found. Just a quick parenthetical like "(The file must be at a path relative to the `CARGO_MANIFEST_DIR`)".

Reply:
> The file must be at a path relative to the `CARGO_MANIFEST_DIR`

Perhaps reference that it's the crate root (and the env)?  I'm not sure `CARGO_MANIFEST_DIR` is meaningful to everyone.  (I certainly didn't know about it until it was mentioned)

Reply:
Agreed; I added docs that didn't explicitly reference `CARGO_MANIFEST_DIR` because I didn't know what it was until I started dumping out `std::env::vars()`

Reply:
Nice resolution, thanks